pkgs/development/interpreters/erlang/R19.nix

```diff
@@ -45,6 +45,8 @@ stdenv.mkDerivation rec {
 
   preConfigure = ''
     ./otp_build autoconf
+  '' + optionalString stdenv.isDarwin ''
+    substituteInPlace configure --replace "sw_vers" "/usr/bin/sw_vers"
```

Comment:
I'm saying that the build should probably not use the system version because our stdenv is pure. The result of a build on el capitan *should* be exactly the same as a build on sierra.